packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -90,39 +85,86 @@ module.exports = function (defaults) {
   });
 }
 
-async function embroiderInRepoAddon(project) {
-  await embroider(project);
-  await addInRepoAddon(project, 'fake-addon');
-}
+// async function embroiderInRepoAddon(project) {
+//   await embroider(project);
+//   await addInRepoAddon(project, 'fake-addon');
+// }
 
-async function addInRepoAddon(project, name, version = '0.0.0') {
+async function addInRepoAddon(project, names, version = '0.0.0') {
   project.linkDependency('ember-add-in-repo-tests', {
     baseDir: __dirname,
   });
 
   project.pkg['ember-addon'] = {
-    paths: [`lib/${name}`],
+    paths: [],
   };
 
-  merge(project.files, {
-    lib: {
-      [name]: {
-        'package.json': {
-          name,
-          version,
-          keywords: ['ember-addon'],
-        },
-        'index.js': `module.exports = {
-          name: require("./package").name,
-        };`,
-        tests: {
-          unit: getTestFiles('with-hooks-assert-includes-test.js'),
+  const lib = {};
+
+  names.forEach((name) => {
+    project.pkg['ember-addon'].paths.push(`lib/${name}`);
+    lib[name] = {
+      'package.json': `{
+        "name": "${name}",
+        "version": "${version}",
```

Comment:
Why is this change necessary?

Reply:
This was so we can have the quotes that json expects. Otherwise we get a build error about malformed json.

Reply:
Oh, we should be `JSON.stringfy`ing this instead, not adding template strings with quotes.